.dockerignore

```diff
@@ -0,0 +1,5 @@
+Dockerfile
+Dockerfile-prod
+.dockerignore
+.git
+.gitignore
```

Comment:
Add *.pyc files

Reply:
Done

Reply:
Oh, sorry, my bad. I read this as .gitignore...
No need for .pyc here.